django/contrib/admin/actions.py

```diff
@@ -22,14 +22,14 @@ def delete_selected(modeladmin, request, queryset):
 
     Next, it deletes all selected objects and redirects back to the change list.
     """
-    opts = modeladmin.model._meta
+    opts = modeladmin.target_model._meta
```

Comment:
Oops, that happened during refactoring, PyCharm changed more than I wanted it seems. Thanks!